docs/content/users/extend/additional-services.md

```diff
@@ -133,9 +133,16 @@ Anyone can create an add-on for `ddev get`. See [this screencast](https://www.yo
 
 The `install.yaml` is a simple YAML file with a few main sections:
 
+* `name`: an add-on name.
```

Comment:
While we're here, why don't we add a little detail. 
```suggestion
* `name`: The add-on name. This will be used in `ddev get --remove` and similar commands. If the repository name is `ddev-elasticsearch`, for example, a good `name` will be `solr`. 
```